RecoCTPPS/Configuration/python/recoCTPPS_cff.py

```diff
@@ -1,7 +1,28 @@
 import FWCore.ParameterSet.Config as cms
 
+from RecoCTPPS.TotemRPLocal.totemRPLocalReconstruction_cff import *
+from RecoCTPPS.TotemRPLocal.ctppsDiamondLocalReconstruction_cff import *
+from RecoCTPPS.TotemRPLocal.totemTimingLocalReconstruction_cff import *
+from RecoCTPPS.PixelLocal.ctppsPixelLocalReconstruction_cff import *
+
+from RecoCTPPS.TotemRPLocal.ctppsLocalTrackLiteProducer_cff import ctppsLocalTrackLiteProducer
+from RecoCTPPS.ProtonReconstruction.ctppsProtons_cfi import *
+
 from Geometry.VeryForwardGeometry.geometryRPFromDB_cfi import *
 
-from RecoCTPPS.Configuration.recoCTPPS_sequences_cff import *
+# TODO: remove these lines once LHCInfo available in DB
+from CalibPPS.ESProducers.ctppsLHCInfo_cff import *
+ctppsProtons.lhcInfoLabel = ctppsLHCInfoESSource.label
+
+# TODO: remove these lines once optical functions available in DB
+from CalibPPS.ESProducers.ctppsOpticalFunctions_cff import *
+ctppsInterpolatedOpticalFunctionsESSource.lhcInfoLabel = ctppsLHCInfoESSource.label
```

Comment:
this replacement pattern here is repeated in about 10 config files.
It looks like it is required for all customers of ctppsOpticalFunctions_cff.
The natural solution is to make this change in ctppsOpticalFunctions_cff so that everyone gets what's needed to run and wouldn't have to override it.

With this, I expect that ctppsOpticalFunctions_cff can already include ctppsLHCInfo_cff.
It sounds like the optical funcs will take a while to make it to the DB.
Once the DB is populated, the only include of ctppsOpticalFunctions_cff can even stay in the client configs; this file content can be just made empty once everything is actually in the DB

Reply:
Thanks for the suggestion, implemented in c1ae6617852.